packages/nimble-components/src/breadcrumb-item/styles.ts

```diff
@@ -75,9 +75,6 @@ export const styles = css`
     slot[name='separator'] svg {
         width: ${iconSize};
         height: ${iconSize};
-    }
-
-    slot[name='separator'] path {
         fill: ${placeholderFontColor};
```

Comment:
The updated svg to ts version had a different svg structure, updated the fill to the top-level svg since fill is inheritable (so all children get the same fill).